### PR TITLE
XIVY-9123 News about azure ad

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

File: src/app/pages/news/9.4/05-azure-ad.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
-Add more meta information to your process elements to ease finding or calling them.
+With Azure Active Directory, Axon Ivy is now able to support its first cloud-based Identity Provider. This means that your employees can simply use the well-known Microsoft login in order to access the Axon Ivy Platform.
 
-- __Tags__: Tag your process elements to easily find them in your projects. Some tags like `Deprecated`, `Connector` or `Demo` have a special meaning.
-- __Parameter Description__: Describe the parameters of start elements to give useful hints to the callers.
+- __Simple user management__: Maintain your users in one place, namely Azure Active Directory. Easily synchronize these users with Axon Ivy and make your business processes instantly accessible to the entire organization.
+
+- __User credentials__: Employees do not have to worry about additional user credentials. Access is simple and secure via the Microsoft login.
+
+- __OAuth 2.0__: With Microsoft login, the industry-standard OAuth 2.0 protocol is used for authorization. Consequently, Axon Ivy is no longer aware of these users' passwords and secure delegated access is granted.
```
Comment (alexsuter) [inline src/app/pages/news/9.4/05-azure-ad.md:9]: authroization -> authentication ?!

**Comment by alexsuter** [comment]
I like it. Needs to be reviewed by our head of grammar @phochstr

File: src/app/pages/news/9.4/05-azure-ad.md (hunk 1 above)
Comment (alexsuter) [reply src/app/pages/news/9.4/05-azure-ad.md:9]: > authroization -> authentication ?!

maybe both would be true..

Comment (phochstr) [reply src/app/pages/news/9.4/05-azure-ad.md:9]: First, we need to authenticate so we know who is accessing the system.
Then, we have to find out what he is allowed to do - authorize.

**Review by phochstr — APPROVED** [review]

File: src/app/pages/news/9.4/05-azure-ad.md (hunk 1 above)
Comment (ivy-lmu) [reply src/app/pages/news/9.4/05-azure-ad.md]: Thanks @phochstr for illuminating

**Pull request merged**